R/teal_reporter.R

```diff
@@ -142,21 +146,38 @@ TealSlicesBlock <- R6::R6Class( # nolint: object_name_linter.
       private$teal_slices <- content
       invisible(self)
     },
-    #' @description Create the `RcodeBlock` from a list.
-    #' @param x (named `list`) with two fields `c("text", "params")`.
-    #' Use the `get_available_params` method to get all possible parameters.
+    #' @description Create the `TealSlicesBlock` from a list.
+    #'
+    #' @param x (`named list`) with two fields `text` and `style`.
+    #' Use the `get_available_styles` method to get all possible styles.
+    #'
     #' @return `self`, invisibly.
+    #' @examples
+    #' TealSlicesBlock <- getFromNamespace("TealSlicesBlock", "teal")
+    #' block <- TealSlicesBlock$new()
```

Comment:
`getFromNamespace` not required as `TealSlicesBlock` is exported

```suggestion
    #' block <- TealSlicesBlock$new()
```